strongbox-commons/src/main/resources/application.yaml

```diff
@@ -34,16 +34,23 @@ strongbox:
     database:
       path: ${strongbox.vault}/db
 logging:
-  dir: strongbox-vault/logs
-  config:
-    file: strongbox/etc/logback-spring-debug.xml
+  debug: false
+  console:
+    enabled: true
```

Comment:
I just tried this with the `strongbox-distribution` by executing `./bin/strongbox console`, but the only thing the distribution output was:

```
 ./bin/strongbox console
Running Strongbox: Distribution...
wrapper  | --> Wrapper Started as Console
wrapper  | Launching a JVM...
jvm 1    | Wrapper (Version 3.2.3) http://wrapper.tanukisoftware.org
jvm 1    |   Copyright 1999-2006 Tanuki Software, Inc.  All Rights Reserved.
jvm 1    | 
jvm 1    | 
jvm 1    |   .   ____          _            __ _ _
jvm 1    |  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
jvm 1    | ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
jvm 1    |  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
jvm 1    |   '  |____| .__|_| |_|_| |_\__, | / / / /
jvm 1    |  =========|_|==============|___/=/_/_/_/
jvm 1    |  :: Spring Boot ::        (v2.1.5.RELEASE)
jvm 1    | 
```

I had to specifically add `STRONGBOX_LOG_CONSOLE_ENABLED=true ./strongbox-distribution/target/strongbox-1.0-SNAPSHOT/bin/strongbox console` for it to start actually outputting the logs. Based on this configuration file, shouldn't `strongbox-distribution` automatically output the logs?

Reply:
:+1:

Reply:
yeah, console logs disabled by default for distribution as we discussed, so when you use `/bin/strongbox start` there will be no excess wrapper logs, file logs are placed in `strongbox-vault/logs`. For `./bin/strongbox console` it will be the same. 
If we want to have console logs enabled for `strongbox-vault/logs` then probably it should be done within followup task.

Reply:
I see - we'll have to add a follow-up task for this. For now it can be used in Docker with an env variable and it works just fine. Lets merge this baby! :D